core/execution.go

```diff
@@ -223,62 +222,90 @@ func StaticCall(env vm.Environment, caller vm.ContractRef, addr common.Address,
 
 // Create creates a new contract with the given code
 func Create(env vm.Environment, caller vm.ContractRef, code []byte, gas, gasPrice, value *big.Int) (ret []byte, address common.Address, err error) {
-	// Depth check execution. Fail if we're trying to execute above the limit.
+	nonce := env.Db().GetNonce(caller.Address())
```

Comment:
Small detail but nonce is retrieved twice unnecessarily, once here and once in the create() function. If others are fine with this I am too but logic can be refactored a bit to not have this happen